payu/experiment.py

```diff
@@ -138,6 +138,9 @@ def init_models(self):
 
         submodels = self.config.get('submodels', [])
 
+        for sm in submodels:
```

Comment:
```suggestion
        # Inject information about required dynamically loaded libraries into submodel configuration
        for sm in submodels:
```
Maybe just add a comment here to help the (code) reader

Reply:
Thanks, I've added it in